src/server/wtr-config.js

```diff
@@ -193,9 +194,14 @@ export class WTRConfig {
 		if (!Array.isArray(browsers)) throw new TypeError('browsers must be an array');
 
 		return browsers.map((b) => playwrightLauncher({
-			concurrency: b === 'firefox' ? 1 : undefined, // focus in Firefox unreliable if concurrency > 1 (https://github.com/modernweb-dev/web/issues/238)
+			concurrency: b === 'firefox' || this.#cliArgs.open ? 1 : undefined, // focus in Firefox unreliable if concurrency > 1 (https://github.com/modernweb-dev/web/issues/238)
 			product: BROWSER_MAP[b],
-			createBrowserContext: ({ browser }) => browser.newContext({ deviceScaleFactor: 2, reducedMotion: 'reduce' })
+			createBrowserContext: ({ browser }) => browser.newContext({ deviceScaleFactor: 2, reducedMotion: 'reduce' }),
+			launchOptions: {
+				headless: !this.#cliArgs.open,
+				devtools: false,
+				slowMo: this.#cliArgs.slowmo || 0
```

Comment:
Maybe I'm missing it, but I don't see any code that actually does anything when `slowMo` is set?

Reply:
It's built into playwright so it slows down operations, though I'm sure what all because they still go pretty fast.

Reply:
Aha makes sense!